site/docs/v1/tech/themes/helpers.adoc

```diff
@@ -161,3 +161,64 @@ The `_helpers.ftl` template provides a couple of macros that help render form el
 * `button`
 ** This macro renders a button that can be used to submit a form. The FusionAuth version of this macro includes an icon and the button text.
 
+== Captcha
+
+The `_helpers.ftl` template provides a macro to embed link:/docs/v1/tech/core-concepts/tenants#captcha-settings[captcha challenges] into your templates. This is
+
+* `captchaBadge`
+** Macro that adds a captcha badge to the template. See link:/docs/v1/tech/themes/template-variables/[template variables] for more information on the template variables. The macro's parameters are:
+
+=== Parameters
+[.api]
+[field]#showCaptcha# [type]#[Boolean]#::
+This determines whether or not to show the captcha badge. Typically supplied by the `tenant.captchaConfiguration.captchaMethod` template variable.
+
+[field]#captchaMethod# [type]#[String]#::
+This is the type of captcha to use. Typically supplied by the `showCaptcha` template variable. Valid values are:
++
+include::/docs/v1/tech/shared/_recaptcha-values.adoc[]
+
+[field]#siteKey# [type]#[String]#::
+The `data-sitekey` value to use for the captcha. Typically supplied by the `tenant.captchaConfiguration.siteKey` template variable. Required if not using `GoogleRecaptchaV3`.
+
+=== Invisible reCAPTCHA
+If you wish to enable an link:https://developers.google.com/recaptcha/docs/invisible[invisible reCAPTCHA] element you may do so by adding the `data-size` and `data-callback` attributes on the tag with the `g-recaptcha` class. In versions >= 1.46.0 these attributes will be present but commented out.
+
+[source,html]
+.Invisible tag
+----
+[#if captchaMethod == "GoogleRecaptchaV2"]
+      <div class="g-recaptcha" data-sitekey="${siteKey!''}"
+        data-size="invisible"
+        data-callback="reCaptchaV2InvisibleCallback"
+      ></div>
+ ...
+----
+
+[NOTE.since]
```

Comment:
I think the way that things are rendered in the doc with this note, the code examples below, and the one line paragraphs makes it hard to follow what is expected here.

It might be worth splitting this into two sub-sections: one for 1.46+ that is just "find this tag and uncomment a couple of lines and you're done" and another for earlier versions that walks through adding attributes to the tag and then modifying JS.

Reply:
Here's an example of post/pre: https://fusionauth.io/docs/v1/tech/tutorials/key-rotation#determining-key-age

Reply:
Put the workaround in the issue and just made this a note to look there